src/api/app/services/notified_projects.rb

```diff
@@ -0,0 +1,26 @@
+class NotifiedProjects
+  def initialize(notification)
+    @notification = notification
+    @notifiable = @notification.notifiable
+  end
+
+  def call
+    return [] unless @notifiable
```

Comment:
`@notification.notifiable` will return `nil` if:
- `@notification.notifiable_type` is `nil`, or
- `@notification.notifiable_id` is `nil`, or
- `@notification.notifiable_type` and`@notification.notifiable_id` are both `nil`

The code `return [] unless @notifiable` catches all these cases. But maybe it is not so readable.

And true, `Project.none` is the right thing to return. Patch applied.

Reply:
```
irb(main):003:0> BsRequest.first.notifications
=> #<ActiveRecord::Associations::CollectionProxy []>
irb(main):004:0> puts "foo" unless BsRequest.first.notifications
=> nil
```

Reply:
> `@notification.notifiable` will return `nil` if:
> 
>     * `@notification.notifiable_type` is `nil`, or
> 
>     * `@notification.notifiable_id` is `nil`, or
> 
>     * `@notification.notifiable_type` and`@notification.notifiable_id` are both `nil`
> 
> 
> The code `return [] unless @notifiable` catches all these cases. But maybe it is not so readable.
> 
> And true, `Project.none` is the right thing to return. Patch applied.

Just for the record, it will also return nil if `@notification.notifiable_id` points to a non existent record...